client/src/index.test.ts

```diff
@@ -264,7 +264,7 @@ async function expectRequest(
   users: RestRepository<User>,
   expectedRequest: ExpectedRequest,
   callback: () => void,
-  response?: unknown
+  response?: any
```

Comment:
Now that ``request`` is a generic, it's hard to infer the type here. Moving to any although it should be possible to fight it.